_pages/docs/matching-options.md

```diff
@@ -0,0 +1,159 @@
+    </expand>
+    <expand property-name="last-name" algorithm-ref="dbl-metaphone" weight="8">
+      <dictionary>name-dictionary.xml</dictionary>
+      <!--defaults to 100 distance -->
+    </expand>
+    <reduce algorithm-ref="std-reduce" weight="4">
+      <all-match>
+        <property>last-name</property>
+        <property>addr1</property>
+      </all-match>
+    </reduce>
+  </scoring>
+  <thresholds>
+    <threshold above="30" label="Possible Match"/>
+    <threshold above="50" label="Likely Match" action="notify"/>
+    <threshold above="68" label="Definitive Match" action="merge"/>
+    <!-- below 25 will be NOT-A-MATCH or no category -->
+  </thresholds>
+  <tuning>
+    <max-scan>200</max-scan>
+  </tuning>
+</options>
+```
+
+### Property Definitions
+
+The `property-defs/property` elements identify the elements that will be used to
+find matches for a document. Use the `namespace` and `localname` attributes to
+specify an XML element or JSON property. The `name` attribute acts as a nickname
+for this property in the rest of the configuration.
+
+### Algorithms
+
+The `algorithms/algorithm` elements list additional algorithms you can use to
+compare property values. The default approach is comparing for equality. To set
+up matching with your custom code, add an `algorithm` element with attributes
+`name`, `function`, `namespace`, and `at`. The `name` attribute is used to refer
+to this algorithm later in the configuration. The other three attributes are
+used to find the code.
+
+### Scoring
+
+Matches are based on how many properties match, what algorithms are used to
+determine matches, and the weights placed on the properties. See the How Scoring
+Works section for more information about how match scores are calculated.
+
+#### `add` elements
+
+This section really drives the matching process. For each `add` element, the
+matcher will add this element or property to the query used to find other
+documents that match the initial document. The `property-name` attribute must
+match the `name` attribute of a `property` element defined under
+`property-defs`. The `weight` attribute is the number of points awarded to
+a potential match if the values match.
+
+#### `expand` elements
+
+To apply a different matching strategy, add an `extend` element. The
+`property-name` element must match the `name` attribute of a `property` element
+defined under `property-defs`. The `algorithm-ref` attribute must match the
+`name` attribute of an `algorithm` element under `algorithms`. Sub-elements
+under the `extend` element depend on the implementation of the function.
+
+#### `reduce` elements
+
+In some cases, a combination of matching properties may suggest a match when
+there shouldn't be one. For instance, two Person records have the same family
+name, same street address, city, and zip code. That might be enough points to
+trigger a match, but if the given names are different, it's probably relatives
```

Comment:
it's probably relatives...

**objection... speculative. lol.**


maybe something like...


_Consider two relatives living together. When matched, two Person records have the same family name, same street address, city, and zip code. That might be enough points to trigger a match even though the two given names differ._